README.md

```diff
@@ -19,24 +15,37 @@ In this workshop we'll be building on what we learnt in the [pg walkthrough](htt
 - Create a superuser with a password by typing `CREATE USER [the new username] WITH SUPERUSER PASSWORD '[the password of the database]';` (the password needs to be in quotes, otherwise you get an error).
 - Change ownership of the database to the new user by typing `ALTER DATABASE [name of the database] OWNER TO [the new username];`
 - Add a `config.env` file and add the database's url in this format:
-`DB_URL = postgres://[username]:[password]@localhost:5432/[database]`.
+`DB_URL = postgres://[username]:[password]@localhost:5432/[database]`. The database name needs to be in lower case.
 
 ### Task 1
 
-In `src/handler.js` you'll find a function that deals with calls to the `/users` endpoint. The data is currently coming from the `static.js` file. You can change the data in `static.js` and check the results.
+In `src/router.js` you'll find a function that deals with calls to the `/users` endpoint. The data is currently coming from the `handlers/static.js` file. You can add your own names and locations in `handlers/static.js`. Once you've done this, refresh the page and check the results.
```

Comment:
If we've moved `static.js` back out of `handlers` then these references need to be updated